### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by Eccb7**
In this project, I have:

- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Added my own styles to improve the look and feel
  - I have used React basic styling (inline styles, importing CSS, and pre-processed CSS files).
- For every page, I have created a <Route>.
- Made sure that navigation links for all 3 routes are displayed on each page.
- Used [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,7 +1,52 @@
...
+
+.navbar-links li a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.navbar-links li a:hover {
+  background-color: #555;
+  padding: 5px;
+  border-radius: 5px;
```
Comment (elisha2kyakpo1) [inline src/App.css:51]: - [ ] Great job adding stylings to keep the pages good, but kindly follow the layout of the given wireframe. You can add colors, fonts, nominations, etc to the app, but keeping the original design is a must. I have noticed that even the calculator design is different from the one you were given in the previous milestone, is there a reason as to why you did this?

design

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Eccb7, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ Quotes are displayed👏
👏 The app works as expected.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (Eccb7) [reply src/App.css:51]: Hi @elisha2kyakpo1 
Hope you are well. 
Thanks for the review though there's this I clearly don't get. Well There's the wireframe but again we're told to (Add your own styles to improve the look and feel)

Well, once again thanks.

**Review by joel-kalema — APPROVED** [review]
Hi @Eccb7 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 


## Highlights
Your design looks good 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**